apps/reports/src/components/ViewEditReport.vue

```diff
@@ -29,6 +29,18 @@
       }}<IconAction v-if="canEdit" icon="pencil-alt" @click="edit = true" />
     </h2>
     <MessageError v-if="error">{{ error }}</MessageError>
+    <div v-if="parameterInputs">
```

Comment:
I suggest making it a bit more friendly when no parameter is set. ( initial condition )

I am getting a bit error `Error: Exception while fetching data (/_reports) : Retrieve of report '1' failed : Query expects parameter 'name': {"response":{"errors":[{"message":"Exception while fetching data (/_reports) : Retrieve of report '1' failed : Query expects parameter 'name'"}],"status":400,"headers":{"map":{"content-type":"application/json","date":"Tue, 13 Jun 2023 09:22:56 GMT","server":"Jetty(9.4.48.v20220622)","transfer-encoding":"chunked"}}},"request":{"query":"query report($parameters:[MolgenisSettingsInput]) {_reports(id:1,parameters:$parameters,limit:5,offset:0){data,count}}","variables":{"parameters":[]}}}`